target/linux/ramips/dts/mt7621_mercusys_mr70x.dts

```diff
@@ -0,0 +1,138 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "mercusys,mr70x", "mediatek,mt7621-soc";
+	model = "Mercusys MR70X";
+
+	aliases {
+		led-boot = &led_orange;
+		led-failsafe = &led_orange;
+		led-running = &led_green;
+		led-upgrade = &led_green;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_green: dome_green {
+			label = "green:dome";
+			gpios = <&gpio 3 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_orange: dome_orange {
+			label = "orange:dome";
+			gpios = <&gpio 4 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	// keys {
+	// 	compatible = "gpio-keys";
+
+	// 	reset {
+	// 		label = "reset";
+	// 		gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
+	// 		linux,code = <KEY_RESTART>;
```

Comment:
why is this commented? Is this not working?

Reply:
It's just copied part but I haven't tested yet

Reply:
resolved.